projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -253,6 +253,11 @@ export interface GridColumn<R> {
      * Whether to show the column as sortable. Defaults to true
      */
     sortable?: boolean;
+
+    /**
+     * The width of this single column in pixels.
+     */
+    width?: number;
```

Comment:
This should not be hardcoded to pixels, people may want a different unit such as em, en, percentage... https://developer.mozilla.org/en-US/docs/Web/CSS/length It should be a string